### PR TITLE
update example nxs reference.

### PR DESCRIPTION
**Pull request by RubelMozumder**

### CODE REVIEW

**Comment by lukaspie** [comment]
Can you check here in the CI/CD that it passes with the respective pynxtools branch?

File: .github/workflows/backward_compatibility.yml [hunk 1]
```diff
@@ -12,7 +12,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        pynxtools_versions: ["master"]
+        # pynxtools_versions: ["master", "basiceln_filename"]
+        pynxtools_versions: ["basiceln_filename"]
```
Comment (RubelMozumder) [inline .github/workflows/backward_compatibility.yml:16]:
```suggestion
        pynxtools_versions: ["master"]
```
Before merge!

File: .github/workflows/backward_compatibility.yml [hunk 2]
```diff
@@ -31,7 +32,7 @@ jobs:
           pip install .[dev]
       - name: Install pynxtools version ${{ matrix.pynxtools_versions }}
         run: |
-         if [ "${{ matrix.pynxtools_versions }}" == "master" ]; then
+         if [ "${{ matrix.pynxtools_versions }}" == "basiceln_filename" ]; then
```
Comment (RubelMozumder) [inline .github/workflows/backward_compatibility.yml:35]:
```suggestion
         if [ "${{ matrix.pynxtools_versions }}" == "master" ]; then
```
before merge!

**Review by lukaspie — APPROVED** [review]
LGTM, please revert changes to `.github/workflows/backward_compatibility.yml` before merging.

**Pull request merged**